### DISCUSSION TITLE
RSS feed for new / changed casks

### DISCUSSION THREAD

**Discussion by mattpackwood** [Homebrew/discussions#2496] [Casks]
Maybe this has been answered before but is there an RSS or e-mail feed of latest casks / formula?

**Comment by neutric** [comment]
As indicated in https://github.com/Homebrew/discussions/discussions/1707, there is https://github.com/Homebrew/homebrew-cask/commits/master.atom. Not sure how useful that is, though.

What exactly are you after?

Just in case you want to catch up on which new apps you can install with the ease of Homebrew, maybe something like

```
% cd $(brew --repository homebrew/cask)
% brew info --cask $(git diff --name-only --diff-filter=A HEAD~1000 HEAD | ggrep -Po 'Casks/\K[^.]*')
```
does the trick. For `ggrep`, do `brew install grep`. The above does a `brew info` on all files added within the last 1000 commits. All you have to do then is take note of `git show HEAD` and use its output instead of `HEAD~1000` when you run the above the next time.

**Comment by neutric** [comment]
@mattpackwood Care to respond?

**Comment by mattpackwood** [comment]
@neutric Sorry, I missed your response.  I will "check it out"

**Comment by neutric** [comment]
No problem and feel free to ask further questions. If satisfied, please mark this as answered.
